test/Altinn.AccessManagement.Tests/Data/blobs/input/app_org1_app1/50002203/p50005545/delegationpolicy.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<xacml:Policy PolicyId="urn:altinn:policyid:4" Version="1.0" RuleCombiningAlgId="urn:oasis:names:tc:xacml:3.0:policy-combining-algorithm:deny-overrides" xmlns:xacml="urn:oasis:names:tc:xacml:3.0:core:schema:wd-17">
+  <xacml:Description>Delegation policy containing all delegated rights/actions from 50002203 to 50005545, for the resource app_org1_app1; </xacml:Description>
+  <xacml:Target />
+  <xacml:Rule RuleId="45453412-efea-4238-8887-6256d8d1727f" Effect="Permit">
+    <xacml:Description>Delegation of a right/action from 50002203 to 20000095, for the resource: app_org1_app1, by user; 20000095</xacml:Description>
+    <xacml:Target>
+      <xacml:AnyOf>
+        <xacml:AllOf>
+          <xacml:Match MatchId="urn:oasis:names:tc:xacml:1.0:function:string-equal">
+            <xacml:AttributeValue DataType="http://www.w3.org/2001/XMLSchema#string">50002203</xacml:AttributeValue>
+            <xacml:AttributeDesignator AttributeId="urn:altinn:partyid" Category="urn:oasis:names:tc:xacml:1.0:subject-category:access-subject" DataType="http://www.w3.org/2001/XMLSchema#string" MustBePresent="false" />
+          </xacml:Match>
+        </xacml:AllOf>
+      </xacml:AnyOf>
+      <xacml:AnyOf>
+        <xacml:AllOf>
+          <xacml:Match MatchId="urn:oasis:names:tc:xacml:1.0:function:string-equal">
+            <xacml:AttributeValue DataType="http://www.w3.org/2001/XMLSchema#string">app_org1_app1</xacml:AttributeValue>
+            <xacml:AttributeDesignator AttributeId="urn:altinn:resource" Category="urn:oasis:names:tc:xacml:3.0:attribute-category:resource" DataType="http://www.w3.org/2001/XMLSchema#string" MustBePresent="false" />
```

Comment:
In the delegation policy for an App delegation the resource part of the policy is stored as two xacml:Match attributes: `urn:altinn:org` and `urn:altinn:app`

Reply:
added them now